cm_sdk/models/__init__.py

```diff
@@ -0,0 +1,266 @@
+from enum import Enum
+import json
+from json import JSONEncoder
+import re
+import logging
+
+__author__ = 'e0d'
+
+
+class StringUtil(object):
```

Comment:
Also, why all these static methods in a class? Why not just top-level functions?